### PR TITLE
Autostart feature

### PR DESCRIPTION
**Pull request by Psychomantis71**
Added as seperate PR.

### CODE REVIEW

File: src/factorio_server.go [hunk 1]
```diff
@@ -43,6 +43,27 @@ func randomPort() int {
...
+	}
+	if FactorioServ.Port == 0 {
+		FactorioServ.Port = 34197
+	}
+	FactorioServ.Savefile = "Load Latest"
+
+	err = FactorioServ.Run()
+
+	if err != nil {
+		log.Printf("Error starting Factorio server: %+v", err)
```
Comment (jannaahs) [inline src/factorio_server.go:61]: I guess the `%+v` is a typo.
```suggestion
		log.Printf("Error starting Factorio server: %v", err)
```

Comment (knoxfighter) [reply src/factorio_server.go:61]: from the golang docs:
```
%v	the value in a default format
	when printing structs, the plus flag (%+v) adds field names
```

Comment (knoxfighter) [reply src/factorio_server.go:61]: We use `%v`, `%+v` and `%s` already in the code, so i dont see a problem there. Also this is only something that is printed in the logs, so i dont see a problem there.

**Comment by knoxfighter** [comment]
Thank you very much. I merged it know :)

**Pull request merged**